nml/global_constants.py

```diff
@@ -59,6 +59,8 @@ def constant_number(name, info, pos):
     "CC_POWDERIZED"   : 11,
     "CC_NON_POURABLE" : 12,
     "CC_NEO_BULK"     : 12,
+    "CC_POTABLE"      : 13,
+    "CC_NON_POTABLE"  : 14,
```

Comment:
I think these don't belong in this PR?

Reply:
No, I failed to branch something correctly, will adjust now.